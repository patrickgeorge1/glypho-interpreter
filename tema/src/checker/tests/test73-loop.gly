"""@oVVoTT#T:::{oKKKVwVVuu":L\e\PsPS{g24$eee