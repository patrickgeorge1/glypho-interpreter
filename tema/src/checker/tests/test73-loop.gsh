i[>io<1-+]o
