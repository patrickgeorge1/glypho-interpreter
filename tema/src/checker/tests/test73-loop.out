3246
2134
6745
2143
675
23
76
-2
-54
2341
0
